cmd/chisel/cmd_cut.go

```diff
@@ -18,10 +13,13 @@ var shortCutHelp = "Cut a tree with selected slices"
 var longCutHelp = `
 The cut command uses the provided selection of package slices
 to create a new filesystem tree in the root location.
+
+By default it fetches the slices for the latest Ubuntu
+version, unless the --release flag is used.
```

Comment:
Is that true?   Isn't to actually to fetch slices for the same Ubuntu release as the current host?

Reply:
You are completely right, changed in https://github.com/canonical/chisel/pull/99/commits/d124e2503566c90d7702f9db1f42ac7b938cb283.